### DISCUSSION TITLE
How install addon in air-gapped environment？

### DISCUSSION THREAD

**Discussion by wanshicheng** [kubevela/kubevela#2583] [Q&A]
How to point the address of helm to our private repo? And How define other resources like that?

**Comment by chivalryq** [comment]
IMO this can be solved by better addon argument mechanism, so that more requirements can be fulfilled. Like kustomize style /cc @zzxwill @hongchaodeng

**Reply by zzxwill** [reply]
Correct. Besides Charts, we should also consider docker images.
